bin/metababel

```diff
@@ -5,13 +5,18 @@ require 'optparse'
 require 'erb'
 require 'metababel'
 require 'fileutils'
+require 'set'
```

Comment:
https://www.rubyguides.com/2019/08/ruby-uniq-method/ `uniq` so to need for `to_set`